common/lib/dependabot/clients/bitbucket.rb

```diff
@@ -160,7 +160,8 @@ def get(url)
           url,
           user: credentials&.fetch("username", nil),
           password: credentials&.fetch("password", nil),
-          idempotent: true,
+          # Setting to false to prevent Excon retries, use BitbucketWithRetries for retries.
+          idempotent: false,
```

Comment:
👍🏻  I'm good with the approach of making this version of the client _really_ not retry to be consistent with the naming of `BitbucketWithRetries`.